lighthouse-core/runner.js

```diff
@@ -301,9 +301,7 @@ class Runner {
   static getArtifactRuntimeError(artifacts) {
     for (const possibleErrorArtifact of Object.values(artifacts)) {
       if (possibleErrorArtifact instanceof LHError && possibleErrorArtifact.lhrRuntimeError) {
-        const errorMessage = possibleErrorArtifact.friendlyMessage ?
-            `${possibleErrorArtifact.friendlyMessage} (${possibleErrorArtifact.message})` :
```

Comment:
this concatenation is now directly handled by gather runner now, right? was that the only case we fell into this path?

Reply:
Yeah, in gather runner. And since this was only introduced in #6051, there's nothing depending on it. `message` here is the error code, so it seemed redundant to keep it when the code is sitting right next to it on the object. I can add it back if we want to match the other place(s?) we put together an error string out of an lherror

Reply:
Nah, sounds good as-is 👍